.github/dependabot.yml

```
@@ -0,0 +1,12 @@
+version: 2
+
+updates:
+  - package-ecosystem: npm
+    directory: "/"
+    schedule:
+      interval: weekly
```

Maybe we can make some groups to avoid having too much PR for the same namespace deps.
```suggestion
      interval: weekly
    groups:
      babel-dependencies:
        patterns:
          - "*babel*"
      fortawesome-dependencies:
        patterns:
          - "*fortawesome*"
      electron-dependencies:
        patterns:
          - "*electron*"
      rollup-dependencies:
        patterns:
          - "*rollup*"
```

---

:+1: